pkg/kn/commands/source/container/container.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/sources/v1alpha2"
+	clientv1alpha2 "knative.dev/eventing/pkg/client/clientset/versioned/typed/sources/v1alpha2"
+)
+
+// NewContainerSourceCommand for managing ApiServer source
+func NewContainerSourceCommand(p *commands.KnParams) *cobra.Command {
+	containerSourceCmd := &cobra.Command{
+		Use:   "container COMMAND",
```

Comment:
Can we explain what `COMMAND` can do in the description?

Reply:
Sure. Add details of COMMAND.